posts/views.py

```diff
@@ -28,6 +29,20 @@ class PostDetailView(DetailView):
     context_object_name = 'post'
 
 
+class PostDeleteView(LoginRequiredMixin, UserPassesTestMixin, DeleteView):
+    model = Post
+    # Django's default template name - posts/post_confirm_delete.html
+    context_object_name = 'post'
+    success_url = '/'
+
+    def test_func(self):
+        post = self.get_object()
+        if self.request.user == post.author:
+            return True
+        else:
+            return False
```

Comment:
```suggestion
        return self.request.user == post.author:
```

Reply:
Got you, fixed it here and also in the update view. 👍